src/styles/card.css

```diff
@@ -0,0 +1,98 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+  
+.card_img {
+  width: 160px;
+  height: 134px;
+  margin-top: 32px;
+}
+  
+.card_apple {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+  text-decoration: none;
+  padding: 0 16px;
+  margin-top: 40px;
+}
+  
+.card_code {
+  display: flex;
+  align-items: flex-start;
+  width: 166px;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  margin-top: 4px;
+}
+  
+.card_block {
+  display: flex;
+  justify-content: space-between;
+  width: 166px;
+  margin-top: 16px;
+}
+  
+.card_reviews {
+  display: flex;
+  align-items: flex-end;
+  text-align: right;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #060b35;
```

Comment:
create variable with a repeating color and use variable instead

Reply:
fix previous comment `create variable with a repeating color and use variable instead`